packages/fxa-payments-server/src/components/PaymentForm/index.tsx

```diff
@@ -190,18 +193,15 @@ export const PaymentForm = ({
       </FieldGroup>
 
       {confirm && plan && (
-        <Checkbox
-          data-testid="confirm"
-          name="confirm"
-          required
-          label={`
-          I authorize Mozilla, maker of Firefox products, to charge my
-          payment method $${formatCurrencyInCents(plan.amount)} per ${
-            plan.interval
-          }, according to payment
-          terms, until I cancel my subscription.
-        `}
-        />
+        <Checkbox data-testid="confirm" name="confirm" required>
```

Comment:
It looks like Prettier did some tidying in this file, but this is the actual change